docs/docs-components/COMPONENT_DATA.js

```diff
@@ -1082,6 +1083,56 @@ const GENERAL_COMPONENT_LIST: $ReadOnlyArray<ListItemType> = [
       figma: null,
     },
   },
+  {
+    svg: <HelpButton />,
+    name: 'HelpButton',
+    aliases: ['HelpButton', 'InfoButton'],
```

Comment:
aliases: ['InfoButton'],

    HelpButton is redundant

Reply:
Done